src/main.rs

```diff
@@ -225,6 +247,44 @@ async fn main() -> anyhow::Result<()> {
     Ok(())
 }
 
+#[cfg(target_arch = "wasm32")]
```

Comment:
We should not duplicate code like this, instead, introduce a function for specific things that are different and switch on the architecture.

Reply:
Two options:
* Use the code I wrote, which changes the `tokio` runtime based on Wasi or native.
* Force native versions of Steward to use single-threaded `tokio`.

Reply:
Discussed offline (it should be possible to use `cfg_attr`)

Reply:
Resolved, `cfg_attr` is used.